swift/ql/test/query-tests/Security/CWE-916/CONSISTENCY/CfgConsistency.expected

```diff
@@ -0,0 +1,5 @@
+deadEnd
+| file://:0:0:0:0 |  ... = ... |
+| file://:0:0:0:0 |  ... = ... |
+| file://:0:0:0:0 |  ... = ... |
+| file://:0:0:0:0 |  ... = ... |
```

Comment:
How do these tests work?  I can only see the `.expected` files in each test directory.

Reply:
Ah I see, thanks.